utils/fft-eval/Makefile

```diff
@@ -0,0 +1,39 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=fft-eval
+PKG_VERSION:=2017-06-28
+PKG_RELEASE=$(PKG_SOURCE_VERSION)
+
+PKG_SOURCE_VERSION:=3cc175570379da172b0b2bcdbb8d2a42f83dad88
+PKG_SOURCE:=FFT_eval-$(PKG_SOURCE_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/simonwunderlich/FFT_eval/tar.gz/$(PKG_SOURCE_VERSION)?
+PKG_HASH:=9d70125fffc83918e6613d5938c4a9925aea07323e09470cf059cf0026e5937c
+PKG_BUILD_DIR:=$(BUILD_DIR)/FFT_eval-$(PKG_SOURCE_VERSION)
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/$(PKG_NAME)
+	SECTION:=utils
+	CATEGORY:=Utilities
+	MAINTAINER:=Nicolas Pace <nicopace@altermundi.net>
+	URL:=https://github.com/simonwunderlich/FFT_eval
+	TITLE:=Evaluates FFT samples from ath9k driver
+	DEPENDS:= +libc @PACKAGE_ATH_DEBUG @PACKAGE_ATH_SPECTRAL
```

Comment:
Am I wrong or is the package not build due to this line? Can't seem to find it in the snapshots repo...

Reply:
https://github.com/openwrt/openwrt/blob/master/package/kernel/mac80211/ath.mk#L97
It's not enabled by default

Reply:
could we enable it or does it have any negative side effects?

Reply:
@diizzyy is it possible to compile it anyway? and then let opkg decide if it's installable?

Reply:
Ofc, just enable ATH_SPECTRAL using menuconfig and it'll popup

Reply:
Sorry, I meant to be available via the official repository

Reply:
No

Reply:
I compared both builds and kmod-ath filesize increases from 12.5K to 13K, is that within the range of a possible activation? Both sysupgrade images seem to keep excatly the same size:

```
25584678 3780 -rw-r--r-- 1 aparcar aparcar 3867328 Feb 27 09:37 openwrt-ar71xx-generic-ap121f-squashfs-sysupgrade.bin_active
25584591 3780 -rw-r--r-- 1 aparcar aparcar 3867328 Feb 27 09:39 openwrt-ar71xx-generic-ap121f-squashfs-sysupgrade.bin_default
```

Reply:
You need to do a PR (main repo) and provide binary sizes, it's not my call so to say ;-)